client/my-sites/checkout/src/payment-methods/credit-card/index.tsx

```diff
@@ -98,3 +67,43 @@ function CreditCardLogos() {
 		</PaymentMethodLogos>
 	);
 }
+
+export function createCreditCardMethod( {
+	store,
+	shouldUseEbanx,
+	shouldShowTaxFields,
+	submitButtonContent,
+	allowUseForAllSubscriptions,
+	existingCardMethods,
+}: {
+	store: CardStoreType;
+	shouldUseEbanx?: boolean;
+	shouldShowTaxFields?: boolean;
+	submitButtonContent: ReactNode;
+	allowUseForAllSubscriptions?: boolean;
+	existingCardMethods?: PaymentMethod[];
+} ): PaymentMethod {
+	const hasExistingCardMethods = existingCardMethods && existingCardMethods.length > 0;
```

Comment:
What if you make _this_ variable as the prop instead of passing all the payment methods? That would simplify the API of `createCreditCardMethod()`: it would only require a boolean rather than an array of a very specific type of object and if we came up with another way to provide that boolean we could switch it out easily.